SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -805,7 +807,7 @@ private boolean checkCampaignRules(int elementCount, String elementName, String
     @SuppressLint("UseSparseArrays")
     protected SwrveConversation _getConversationForEvent(String event) {
         SwrveConversation result = null;
-        SwrveCampaign campaign = null;
+        SwrveMessageCampaign campaign = null;
```

Comment:
Its mostly because of the initial implementation. We brought in SwrveConversations as a copy of SwrveMessages and theres still some tech debt around it.

If the var isn't used however then I'll :hocho: